inmap/cmd/config.go

```diff
@@ -118,8 +118,8 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 			"there has been an error parsing the configuration file: %v\n", err)
 	}
 
-	for i, v := range config.OutputVariables {
-		config.OutputVariables[i] = os.ExpandEnv(v)
+	for k, v := range config.OutputVariables {
+		config.OutputVariables[k] = os.ExpandEnv(v)
```

Comment:
So there was not really a problem with the way you had it, but because we say in the documentation that it can include environment variables, it makes sense to allow the environment variables in both the names and the formulas.